.github/workflows/automatic-check.yml

```diff
@@ -0,0 +1,58 @@
+name: Automatic documentation checks
+
+on:
+  - push
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
```suggestion
  group: ${{ format('{0}-{1}', github.workflow, github.ref_name) }}
```

Reply:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#format is used here.

Reply:
Here I was using the same as in Charmed Aether SD-Core documentation?